### PR TITLE
CHM-394 marine-ebsa report view page

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrate marine-ebsa report view page

### Designs
same as absch , chm project 

### Testing instructions
search-> report type->marine ebsa

### remian issues: ( need solve in the future)
map display  issue
( The map display is incomplete when the page is loaded; if right-click inspect , the map will be displayed completely.）

incomplete map:
image

complete map: 
image
image
image

### CODE REVIEW

File: app/views/forms/view/chm/marine-ebsa.vue [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+                    <div class="row">    
+                        <div class="col-8" v-if="document.recommendedToCopByGovernment">
+                            <label>{{t("ongoingOfficialProcess")}}</label>
+                            <span class="km-value">
+                                <km-term :value="document.recommendedToCopByGovernment" :locale="locale"></km-term>
+                            </span>
+                        </div>
+                        <div class="col-4" v-if="fixDate(document.recommendedToCopByGovernmentOn)">
+                            <label>{{t("date")}}</label>
+                            <div class="km-value">{{fixDate(document.recommendedToCopByGovernmentOn)}}</div>
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/views/forms/view/chm/marine-ebsa.vue]: fixed

File: app/views/forms/view/chm/marine-ebsa.vue [hunk 2]
```diff
@@ -36,12 +36,17 @@
...
                             <ng v-vue-ng:km-link-list v-model:ng-model="document.gisFiles"  ></ng> 
-                        </div>     
-                        <div v-if="mapConfig">  
+                        </div> 
+                    </div>
+                   
+                    <div v-if="mapConfig"> 
+                        <label>{{ t("geoLocation") }} </label>  
+                        <div class="km-value">  
                          <!-- TODO: fix map delay display issue -->
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/marine-ebsa.vue:50]: do we still have any issue to fix with the map?

Comment (liyuanmontreal) [reply app/views/forms/view/chm/marine-ebsa.vue:50]: no , I will remove it

File: app/views/forms/view/chm/marine-ebsa.vue [hunk 3]
```diff
@@ -99,161 +104,193 @@
...
-                    <div :class="[(document.status=='approved')? 'bg-success text-white ':'']">
-                        {{t("areasEbsa")}}
-                    </div> 
-                    <div v-if="document.approvedByCopDecision" >
-                        <div v-if="approvedByCopDecisionDate != undefined" >
-                            <label>{{t("copDecision")}}</label> 
-                            <div class="km-value">{{approvedByCopDecisionDate}}</div> 
-                        </div>
+                <div v-if="document.status=='approved' || document.approvedByCopDecision || document.approvedByCopDecision" >
+                    <div class="card">
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/views/forms/view/chm/marine-ebsa.vue:111]: ok, I will change it

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**